src/components/backend-ai-usersettings-general-list.ts

```diff
@@ -1138,6 +1188,13 @@ export default class BackendAiUsersettingsGeneralList extends BackendAIPage {
               unelevated
               label="${_t('button.Generate')}"
               @click="${this._refreshSSHKeypair}"></mwc-button>
+          <mwc-button
+              unelevated
+              label="${_t('button.EnterManually')}"
+              @click="${() => {
+                this._initManualSSHKeypairFormDialog();
```

Comment:
@yomybaby I just add a function for resetting the input field when opening the dialog since It should be deleted after registration.

Reply:
👏